### PR TITLE
Home Page

### PR DESCRIPTION
**Pull request by AbhishekkGautam**
### Home Page
- Navigation bar
  - Search Box
  - Wishlist Icon
  - Cart Icon
  - User Account Icon
- Hero Banner
- Categories Section
- Featured Products Section
- Footer

### CODE REVIEW

**Comment by pritam-kr** [comment]
Everything looks awesome to me.

**Review by pritam-kr — APPROVED** [review]

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+}
+
+.hero-btn {
+  margin: 2rem 0;
+}
+
+.category-title {
+  font-size: 1.5rem;
+  margin-top: 3rem;
+  margin-bottom: 2rem;
```
Comment (gopalbbharadva) [inline css/main.css:44]: Improvement: It can be margin: 3rem 0 2rem 0

**Review by gopalbbharadva — APPROVED** [review]
Clean UI:)

File: css/main.css (hunk 1 above)
Comment (AbhishekkGautam) [reply css/main.css:44]: Thank you. I'll refactor it.

**Pull request merged**